com.obsproject.Studio.Plugin.Teleport.yaml

```diff
@@ -0,0 +1,56 @@
+id: com.obsproject.Studio.Plugin.Teleport
+branch: stable
+runtime: com.obsproject.Studio
+runtime-version: stable
+sdk: org.freedesktop.Sdk//24.08
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.golang
+build-extension: true
+separate-locales: false
+appstream-compose: false
```

Comment:
then why install the metainfo?

Reply:
I didn't know what this was for, most of the info on making a flatpak revolved around making apps so I copied another flatpak addon's manifest to start and they had this, I've enabled using the metadata.

Reply:
well actually it's supposed to use an older SDK (see my other comment) which mean that appstream-compose shall still be run manually.

Reply:
no, without this flatpak-builder will execute appstreamcli from `org.flatpak.Builder`, the one in the runtime has no effect.